readme.md

```diff
@@ -1,69 +1,22 @@
 # ReaKontrol
-
```

Comment:
This file shouldn't be included in this PR/branch.

Reply:
yeah, I was surprised it got in there. don't even know how to exclude it ;-) such a rookie  I am with git. need to study the documentation better.
... don't be shocked ... the code itself is (despite of style issues) not as terrible